apps/announcements/views.py

```diff
@@ -0,0 +1,33 @@
+from apps.announcements.models import Announcement
+from apps.checkin.views import success_data_jsonify
+from rest_framework.decorators import api_view
+import os
+import datetime
+from fcm_django.models import FCMDevice
+
+APPROVED_NUMBERS = []  # TODO: Add numbers here
+
+
+@api_view(['GET', 'POST'])
+def announcements(request):
+    if request.POST:
+        form = request.POST
+        body = form['Body']
```

Comment:
Not a Django expert, but I believe accessing on request.POST gives us the form-data instead of the body. I really hate using form data so let's put everything in the body as JSON and parse from there.

Reply:
I need to look into the twilio bot more, but for MenloHacks I think it just sent a form body. Since the post on this will never be called directly it doesn't matter too much.

Reply:
@VjunetXuuftofi You're right! Wasn't focused on the fact that this was twilio code at first